### PR TITLE
Add richer status field display for operator table

### PR DESCRIPTION
**Pull request by jcaianirh**
For Jira story: https://jira.coreos.com/browse/CONSOLE-1463

Implement designs here:  https://projects.invisionapp.com/share/7TRU0EYHJNM#/screens/361317782 

cc: @tlwu2013 @jhadvig 

Screen Shot 2019-07-18 at 2 17 31 PM

After adding some logic for state, and conditions:

Screen Shot 2019-08-08 at 12 43 13 PM

Screen Shot 2019-08-08 at 12 40 35 PM

Screen Shot 2019-08-08 at 12 52 39 PM

### CODE REVIEW

**Comment by jcaianirh** [comment]
@alecmerdler could you review this please? Does this fulfill what we want to display for status with conditions?

File: frontend/__tests__/components/operator-lifecycle-manager/operand.spec.tsx [hunk 1]
```diff
@@ -368,3 +372,38 @@ describe(ProvidedAPIPage.displayName, () => {
     expect(wrapper.find(ListPage).props().canCreate).toBe(false);
   });
 });
+
+describe('OperandStatusIconAndText', () => {
+  let wrapper: ShallowWrapper<OperandStatusIconAndTextProps>;
+
+  it('dispalys the correct status and status type for a status value of running', () => {
```
Comment (rhamilto) [inline frontend/__tests__/components/operator-lifecycle-manager/operand.spec.tsx]: typo: "dispalys"

**Comment by beanh66** [comment]
FYI @rileyhuston

**Comment by spadgett** [comment]
It looks like this is replaced by #3121

/close

**Pull request closed without merging**